packages/clerk-js/src/ui/components/SignUp/SignUp.tsx

```diff
@@ -56,7 +56,21 @@ function SignUpRoutes(): JSX.Element {
           />
         </Route>
         <Route path='continue'>
-          <SignUpContinue />
+          <Route index>
+            <SignUpContinue />
+          </Route>
+          <Route
+            path='verify-email-address'
+            canActivate={clerk => !!clerk.client.signUp.emailAddress}
+          >
+            <SignUpVerifyEmail />
+          </Route>
+          <Route
+            path='verify-phone-number'
+            canActivate={clerk => !!clerk.client.signUp.phoneNumber}
+          >
+            <SignUpVerifyPhone />
+          </Route>
```

Comment:
❓ Shouldn't the `index` route be the last item here?

Reply:
Do you mean moving it as the last child? Does that matter?